packages/@lwc/engine-server/src/__tests__/fixtures/styles-escaping-safe/expected.html

```diff
@@ -0,0 +1,9 @@
+<x-styles>
+  <template shadowroot="open">
+    <style type="text/css">
+      [data-foo="'&
+      <>
+        "] {color: black;}[data-foo='"'] {color: white;}
+      </style>
```

Comment:
I'm still trying to figure out how to fix the fixture formatter. Just trust me that the serialized HTML doesn't really look like this.

Reply:
I've struggled with this for quite some time. Other options (e.g. switching to Prettier) had their own downsides. I think for now I'm fine with just having the fixture look a little ugly.

Reply:
Fixed it